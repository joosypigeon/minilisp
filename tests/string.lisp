(print "hello world")         ;; → hello world
(print (+ 5 6))               ;; → 11
(print (cons 1 2))            ;; → (1 . 2)
(print (lambda (x) x))        ;; → (lambda (x) x)
(print (equal? "hello" "hello"))    ;; => #t
(print (equal? "hello" "world"))    ;; => ()
(print (define greet "hi"))
(print greet)                       ;; => "hi"
(print "Done!")              ;; (if you have `print`)