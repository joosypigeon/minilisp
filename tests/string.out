"hello world"
11
(1 . 2)
(lambda (x) (x))
#t
NIL
"hi"
"hi"
"Done!"
